superset-frontend/src/SqlLab/components/ResultSet/index.tsx

```diff
@@ -100,8 +99,8 @@ const MonospaceDiv = styled.div`
 `;
 
 const ReturnedRows = styled.div`
-  font-size: 13px;
-  line-height: 24px;
+  font-size: ${({ theme }) => theme.gridUnit * 3.25}px;
```

Comment:
Please use `theme.typography.sizes`. I know that we don't have font size 13, but you can pick size `s` (12)